src/main/resources/hudson/scm/SubversionSCM/DescriptorImpl/credential-ok.js

```diff
@@ -0,0 +1,3 @@
+window.addEventListener("DOMContentLoaded", () => {
+    document.querySelector(".svn-credential-ok-close").addEventListener("click", () => window.close());
```

Comment:
Elsewhere in your changes, you've been putting one-line lambdas into their own block. I don't care either way, as long as we're consistent within the same file.

```suggestion
    document.querySelector(".svn-credential-ok-close").addEventListener("click", () => {
        window.close();
   });
```